test/data/navigate-to-site.html

```diff
@@ -0,0 +1,16 @@
+<html>
+<head>
+<script>
+function redirectToDestination() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const url = urlParams.get('url');
+    window.location = atob(url);
+}
+</script>
+</head>
+<body>
+<p>Waiting for JS redirect...</p>
+<script>
+setTimeout(redirectToDestination, 100);
```

Comment:
curious why do we need this?

Reply:
That's to simulate a navigation, as opposed to a redirect. Another way I could have done this would have been to include a link and then inject JS to click the link programmatically.

Reply:
100ms is pretty noticeable dealy even for browsertests, so I'd suggest to change if we can avoid it